app/pages/preview/templatePrint.njk

```diff
@@ -0,0 +1,96 @@
+{% extends 'includes/layoutPrint.njk' %}
+{% from 'components/back-link/macro.njk' import backLink %}
+{% from 'components/bc-table-flexible/macro.njk' import bcTableFlexible %}
+
+{% block body %}
+  <div data-test-id="preview-page-print">
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-two-thirds">
+        <h1 data-test-id="preview-page-title" class="nhsuk-u-margin-top-7 nhsuk-u-margin-bottom-3">{{ title }}</h1>
+        <h2 data-test-id="preview-page-description" class="nhsuk-u-font-size-24 nhsuk-u-margin-bottom-6">{{ description }}</h2>
+
+        <h3 data-test-id="order-description-heading" class="nhsuk-u-margin-bottom-3">{{ orderDescriptionHeading }}</h3>
+        <h4 data-test-id="order-description" class="nhsuk-u-font-size-19 nhsuk-u-font-weight-normal">{{ orderDescription }}</h4>
+      </div>
+    </div>
+  
+    <div class="nhsuk-grid-row nhsuk-u-margin-bottom-3">
+      <div class="nhsuk-grid-column-full">
+        <div data-test-id="date-summary-created">
+          <span class="nhsuk-u-font-weight-bold">{{ dateSummaryCreatedLabel }}</span> {{ dateSummaryCreated }}
+        </div>
+      </div>
+    </div>
+
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-full">
+        <div data-test-id="calloff-and-supplier"> 
+          {{ bcTableFlexible({
+            columnInfo: callOffAndSupplierTable.columnInfo,
+            classes: callOffAndSupplierTable.classes,
+            data: callOffAndSupplierTable.items
+          }) }}
+        </div>
+      </div>
+    </div>
+
+    <div class="nhsuk-grid-row nhsuk-u-margin-bottom-3">
+      <div class="nhsuk-grid-column-full">
+        <div data-test-id="commencement-date">
+          <span class="nhsuk-u-font-weight-bold">{{ commencementDateLabel }}</span> {{ commencementDate }}
+        </div>
+      </div>
+    </div>
+
+    <div style='page-break-after:always'></div>
```

Comment:
Would it be better to use a class attribute here rather than an in line style?

Reply:
I'm not sure. This is just styling for the print page to put the next section on a different page. Adding a new style involves creating a scss file and then including that which is lot more involved for something that is just temporary hack for a low cost solution to generating a pdfable page. I didn't want to over-engineer this guy.